ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/NamingStrategy.java

```diff
@@ -17,6 +17,8 @@
  */
 public interface NamingStrategy {
 
+    void initialiseNetwork(Network network);
```

Comment:
Can you rename this method to `initializeNetwork` (with a 'z', which is the US English spelling)?

Reply:
I renamed it